staging/src/k8s.io/cloud-provider/fake/fake.go

```diff
@@ -321,13 +323,20 @@ func (f *Cloud) InstanceMetadata(ctx context.Context, node *v1.Node) (*cloudprov
 	f.addCall("instance-metadata-by-provider-id")
 	f.addressesMux.Lock()
 	defer f.addressesMux.Unlock()
+
+	providerID := ""
```

Comment:
@andrewsykim so I make this default empty providerID instead of inheriting from `node.spec.providerID`
PTAL :)